app/components/ui/PeoplePicker/types.js

```diff
@@ -9,18 +9,16 @@ export const personNamePropType = PropTypes.string
 
 export const affiliationPropType = PropTypes.string
 
-export const affiliationsPropType = PropTypes.arrayOf(affiliationPropType)
-
 export const expertisesPropType = PropTypes.arrayOf(PropTypes.string.isRequired)
 
 export const focusesPropType = PropTypes.arrayOf(PropTypes.string.isRequired)
 
 export const personPropType = PropTypes.shape({
   id: personIdPropType.isRequired,
   name: personNamePropType.isRequired,
-  aff: affiliationsPropType,
-  expertises: expertisesPropType,
-  focuses: focusesPropType,
+  aff: affiliationPropType,
+  expertises: expertisesPropType.isRequired,
+  focuses: focusesPropType.isRequired,
```

Comment:
`expertises` & `focuses` still have `isRequired` properties. Has something changed elsewhere that I'm missing to cast these to empty arrays?

Reply:
The shape of the object we create on the server side will always have empty arrays on

```
research: {
  focuses: []
  expertises: []
}
```

https://github.com/elifesciences/elife-xpub/blob/develop/server/xpub-model/entities/user/helpers/elife-api.js#L23-L24

I think we should change the server-side code so this isn't the case, but if we do that it's prob worth doing in a separate issue.